Mage/src/main/java/mage/abilities/effects/common/ruleModifying/CombatDamageByToughnessAllEffect.java

```diff
@@ -0,0 +1,58 @@
+
+package mage.abilities.effects.common.ruleModifying;
+
+import mage.abilities.Ability;
+import mage.abilities.Mode;
+import mage.abilities.effects.ContinuousEffectImpl;
+import mage.constants.Duration;
+import mage.constants.Layer;
+import mage.constants.Outcome;
+import mage.constants.SubLayer;
+import mage.filter.common.FilterCreaturePermanent;
+import mage.game.Game;
+
+/**
+ * @author TheElk801
+ */
+public class CombatDamageByToughnessAllEffect extends ContinuousEffectImpl {
+
+    private final FilterCreaturePermanent filter;
+
+
+    public CombatDamageByToughnessAllEffect(FilterCreaturePermanent filter) {
+        this(filter, Duration.WhileOnBattlefield);
+    }
+     
+    public CombatDamageByToughnessAllEffect(FilterCreaturePermanent filter, Duration duration) {
+        super(duration, Layer.RulesEffects, SubLayer.NA, Outcome.Detriment);
```

Comment:
Outcome.Neutral?

Reply:
The original had Outcome.Detriment, I wasn't sure what to put there

Reply:
Well the others are neutral, and neutral makes sense because whether it's good or not depends on whether toughness is greater than power. Outcome is only used by the AI so not a big deal really.